### ISSUE TITLE
[Bug]: Stuck at loading when I teleport

### ISSUE THREAD

**Issue by SeniorPollen** [LukeYui/EldenRingSeamlessCoopRelease#53]
### Discord Username

Pollen#3364

### Describe the issue in detail

It has happened already twice. I play with a friend, he enteres my world and everytime I try to walk through Raya´s Lucaria door (first time to access the academy) my game never quits loading forcing me to alt+f4. First time the game loaded, I touched the grace and went to bed. Next day I tried to enter the save and I could not get pass the loading screen. I tried to reinstall the mod but the file was cursed. I was able to start another run without problem so I realised that it was not the mod but the file what was wrong. I started a second run and this time when I teleported the acadamy I got stuck in that loading screen. I relintalled elden ring, upgrade the version of the mod to the last one, I,ve checked any corrupt files in Steam... and nothing. 
I hope you can help me somehow, I would not like to start a third run just to be corrupted again. 
Thank you for reading me.

### Steps to recreate the bug

Get the key from the dragon to access the raya lucaria academy. Go to raya lucaria´s door and open it. As soon as you teleport the screen never stops loading. I was playing with a friend, he entering my world.

### Version

1.2.0 (Latest)

**Comment by LukeYui** [comment]
Hi, thanks for the report.

I'll look into it, though I cannot reproduce the bug.

**Comment by AsherIsrael** [comment]
I'm in a similar situation. I was playing with a group of friends, two of them had the key already, me and one other did not. We collected the key but then left to do other things. Later, on my own, I tried entering the academy and saw a very long loading screen, after which I loaded into the courtyard, but the game lagged heavily and then crashed. Trying to open the game and select "continue" subsequently caused the game to crash during the loading screen. If I load the game instead of continuing it successfully loads with my in the entrance courtyard, but the game still lags heavily and crashes before I can interact with the site of grace or during the loading screen if I try to warp somewhere else. During one crash I had the following error message:
![image](https://user-images.githubusercontent.com/17623022/173715952-50632986-005b-456e-81b0-4034f9b145ed.png)
